Frameworks/Foundation/NSException.cl.cpp

```diff
@@ -54,9 +56,18 @@ extern "C" LONG WINAPI _NSWindowsUnhandledExceptionFilter(struct _EXCEPTION_POIN
             }
 
             if (foundNSException && foundobjc_object) {
+                Microsoft::WRL::ComPtr<IRestrictedErrorInfo> comErrorInfo;
+                ::GetRestrictedErrorInfo(&comErrorInfo); // save this here since the unhandled exception handler might stomp it.
```

Comment:
nit: probably move it closer to if (comErrorInfo) check

---
In reply to: [118055298](https://github.com/Microsoft/WinObjC/pull/2714#discussion_r118055298) [](ancestors = 118055298)